modules/openapi-generator/src/test/java/org/openapitools/codegen/AllGeneratorsTest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech)
+ * Copyright 2018 SmartBear Software
```

Comment:
I think you can remove this line per https://opensource.stackexchange.com/questions/7300/copyright-notice-in-the-file-header-apache-v2-license